wordpress-mobile/gutenberg-mobile
# Stories block: hide/show based on Editor capability settings being set, only after it's been loaded

We observed the settings may not be ready exactly when the `native.render` hooks get run but rather right after that execution cycle. Hence, we're only checking for the actual settings to be loaded by using `setTimeout` without delay

To test:
- verify the tests are now passing with `npm run test src/test/index.js`

Note: I've considered and tried adding unit tests for the specific functionality added but, since we haven't found a way to mock select/dispatch easily yet - will get back at this at a later point.

PR submission checklist:
